tests/pagination/tests.py

```diff
@@ -233,6 +233,21 @@ def test_get_page_hook(self):
         self.assertEqual(page2.previous_page_number(), 1)
         self.assertIsNone(page2.next_page_number())
 
+    def test_page_range_iterator(self):
+        """
+        Tests that the Paginator is an iterator.
+        """
+        iterator = Paginator([1, 2, 3], 2).page_range
+        self.assertTrue(hasattr(iterator, "__iter__"))
+        try:
```

Comment:
Also, this is not good that tests differences between pythons versions are via exceptions.
You should use `six`